.github/workflows/build.yml

```diff
@@ -275,6 +275,27 @@ jobs:
       - name: For each commit, run cargo check (including in fuzz)
         run: ci/check-each-commit.sh upstream/main
 
+  check_release:
+    runs-on: ubuntu-latest
+    env:
+      TOOLCHAIN: 1.57.0
```

Comment:
How do we decide what toolchain to use? I see fuzzing use 1.58.0 and linting 1.47.0.

Reply:
Good question, I now switched to `stable` to avoid giving a specific version.

Reply:
Not quite sure about fuzzing, tbh. `stable` seems fine for `check_release`. We typically only use `nightly` if we need experimental features, IIUC.

Reply:
> `stable` seems fine for `check_release`. We typically only use `nightly` if we need experimental features, IIUC.

Yeah, that's what I assumed.